polyply/src/map_to_molecule.py

```diff
@@ -13,12 +13,9 @@
 # limitations under the License.
 
 import networkx as nx
-from vermouth.graph_utils import (make_residue_graph,
-                                  collect_residues)
 from polyply.src.processor import Processor
-from polyply.src.graph_utils import is_branched
 
-def tag_exclusions(blocks, force_field):
+def tag_exclusions(node_to_block, force_field):
```

Comment:
docstring out of date

Reply:
how why? The docstring is still accurate

Reply:
Docstring still references `blocks`